packages/xrpl/HISTORY.md

```diff
@@ -13,6 +13,7 @@ Subscribe to [the **xrpl-announce** mailing list](https://groups.google.com/g/xr
 * `TransactionStream` model includes `hash` field in APIv2
 * `TransactionStream` model includes `close_time_iso` field only for APIv2
 * Adds `MPTCurrency` type
+* Adds pseudo transactions to `NON_DELEGATABLE_TRANSACTIONS`
```

Comment:
Unneeded since this is the same version containing the original `Delegate` feature